.github/workflows/build.yml

```diff
@@ -56,95 +56,86 @@ jobs:
         path: luac.cross
 
 
-  build_luac_cross_win:
```

Comment:
Nice to see that we will finally have a test for the mingw build

but please do not remove the visualstudio build of the tools.
It can be built with VS and does not need mingw

same for NTest_win.

Maybe call the new sections something with cygwin like NTest_cygwin

Reply:
ok, got it. that's the ones not working currently.
So I will have a look and re add them when I found the solution.
Will start with the work by @marcelstoer on this

Reply:
Maybe the best solution for now

Reply:
I don't remember why I thought handling it that way was a good idea (it wasn't), sorry. We could cherry-pick the `release` commit into the `dev` branch, rebase this one onto `dev` and proceed from there.
@HHHartmann feel free to execute this if you agree.